### DISCUSSION TITLE
Error message: 809

### DISCUSSION THREAD

**Discussion by viciopoli01** [gradescope/autograder_samples#125] [Q&A]
Hello everyone,

We've been using the Autograder for a class over the past few years without issues, but recently we've encountered a new error message:
```
Autodriver: Job exited with status 3
Autodriver: Error executing make at line 533: Resource temporarily unavailable
```
Has anyone dealt with this issue before or have any suggestions on how to resolve it?

Thanks in advance for your help!

**Comment by adam-brucker** [comment]
Check your system’s resource usage (e.g., memory, CPU, disk space).
Retry the operation to see if it was a transient issue.
Investigate line 533 of the make script for the specific command or resource it tried to access.
